lib/features/children/add_member/pages/add_member_form_page.dart

```diff
@@ -0,0 +1,346 @@
+      },
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final formKey = GlobalKey<FormState>();
+    final user = context.read<AuthCubit>().state.user;
+    final currency = NumberFormat.simpleCurrency(
+      name: Util.getCurrencyName(country: Country.fromCode(user.country)),
+    ).currencySymbol;
+
+    return BlocBuilder<AddMemberCubit, AddMemberState>(
+      builder: (context, state) {
+        return Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 20),
+          child: Column(
+            children: [
+              setUpFamilyHeader(),
+              const SizedBox(height: 32),
+              childOrParentSelector(),
+              const SizedBox(height: 10),
+              animate(
+                child: createChildForm(formKey, currency),
+                isVisible: isChildSelected,
+              ),
+              animate(
+                isVisible: !isChildSelected,
+                child: Text(
+                  'Coming soon',
+                ),
+              ),
+              if (View.of(context).viewInsets.bottom <= 0) const Spacer(),
+              continueButton(formKey: formKey, enabled: isChildSelected),
+            ],
+          ),
+        );
+      },
+    );
+  }
+
+  Widget animate({required Widget child, required bool isVisible}) =>
+      AnimatedOpacity(
+          opacity: isVisible ? 1.0 : 0.0,
+          duration: Duration(milliseconds: 200),
+          child: child);
+
+  Widget childOrParentSelector() => Row(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          selectorSegment(
+            title: 'Child',
+            isSelected: isChildSelected,
+            isLeft: true,
+          ),
+          selectorSegment(
+            title: 'Parent',
+            isSelected: !isChildSelected,
+            isLeft: false,
+          ),
+        ],
+      );
+  Widget selectorSegment({
+    required String title,
+    required bool isSelected,
+    required bool isLeft,
+  }) =>
+      GestureDetector(
+        onTap: () {
+          setState(() {
+            isChildSelected = !isChildSelected;
+          });
+        },
+        child: Container(
+          padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 10),
+          decoration: BoxDecoration(
+            color: isSelected
+                ? Theme.of(context).colorScheme.primary
+                : Colors.white,
+            borderRadius: isLeft
+                ? BorderRadius.only(
+                    topLeft: Radius.circular(4),
+                    bottomLeft: Radius.circular(4),
+                  )
+                : BorderRadius.only(
+                    topRight: Radius.circular(4),
+                    bottomRight: Radius.circular(4),
+                  ),
+            border: isSelected
+                ? Border.all()
+                : Border.fromBorderSide(
+                    BorderSide(
+                      color: Colors.grey.shade300,
```

Comment:
Don't we have a custom color for this grey? We different colors for grey, it would be nice to centrelize this

Reply:
`Colors.grey.shade300` is used in the AppTheme to define the border colors (including for the text fields). The textformfield border has to match the hint/label text color, so will update it there instead.